core-services/workflow/src/test/java/org/silverpeas/core/workflow/engine/user/ReplacementTest.java

```diff
@@ -103,15 +103,6 @@ public void createAReplacementToHimself() {
         .during(aPeriod);
   }
 
-  @Test
-  public void saveANewReplacement() {
```

Comment:
Because this is a unit test and this test fails because of the missing true persistence framework. And the persistence is tested by the integration test ReplacementIT